test/common.js

```diff
@@ -72,7 +72,7 @@ var BHYVE_128_FLEX_PACKAGE = Object.assign({}, BHYVE_128_PACKAGE, {
     uuid: '8b3b6796-e5b1-47ce-bb49-ac9dcba7c6eb',
     brand: 'bhyve',
     name: 'sdc_128_flex_bhyve',
-    quota: 50 * 1024, // 50 GiB
+    quota: 20 * 1024, // 50 GiB
```

Comment:
Why is this changing? I'm not 100% sure, but I think some depends on this being 50Gib.

Reply:
I'd bias for keeping this value as-is, but if there's sufficient rationale for reducing the quota I'm not inherently opposed. But in any case the comment should match the value.